Dockerfile

```diff
@@ -25,7 +25,9 @@ ENV AWS_PAGER=""
 
 RUN apk add --no-cache \
   bash \
+  git \
   curl \
+  wget \
```

Comment:
Why do you need wget?

Reply:
Needed wget for the script since curl was having issues when ran in github actions.